src/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer/Microsoft.ML.DnnAnalyzer.csproj

```diff
@@ -12,7 +12,7 @@
     <ProjectReference Include="..\..\Microsoft.ML.TensorFlow\Microsoft.ML.TensorFlow.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(NativeTargetArchitecture)' == 'x64'">
```

Comment:
Maybe worth `<!-- TensorFlow is 64 bit only -->`

Reply:
Agreed. I also think it would be good to document on each `[ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]` what about the test doesn't allow it to be run on x86 processes.

Reply:
Fixed.